bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/converter/ZWaveSwitchAllConverter.java

```diff
@@ -0,0 +1,119 @@
+import org.openhab.binding.zwave.internal.converter.command.BinaryOnOffCommandConverter;
+import org.openhab.binding.zwave.internal.converter.command.ZWaveCommandConverter;
+import org.openhab.binding.zwave.internal.converter.state.BinaryDecimalTypeConverter;
+import org.openhab.binding.zwave.internal.converter.state.BinaryPercentTypeConverter;
+import org.openhab.binding.zwave.internal.converter.state.IntegerOnOffTypeConverter;
+import org.openhab.binding.zwave.internal.converter.state.IntegerOpenClosedTypeConverter;
+import org.openhab.binding.zwave.internal.converter.state.IntegerUpDownTypeConverter;
+import org.openhab.binding.zwave.internal.converter.state.ZWaveStateConverter;
+import org.openhab.binding.zwave.internal.protocol.SerialMessage;
+import org.openhab.binding.zwave.internal.protocol.ZWaveController;
+import org.openhab.binding.zwave.internal.protocol.ZWaveNode;
+import org.openhab.binding.zwave.internal.protocol.commandclass.ZWaveBatteryCommandClass;
+import org.openhab.binding.zwave.internal.protocol.commandclass.ZWaveCommandClass;
+import org.openhab.binding.zwave.internal.protocol.commandclass.ZWaveSwitchAllCommandClass;
+import org.openhab.binding.zwave.internal.protocol.event.ZWaveCommandClassValueEvent;
+import org.openhab.core.events.EventPublisher;
+import org.openhab.core.items.Item;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/***
+ * ZWaveSwitchAllConverter class. Converter for communication with the 
+ * {@link ZWaveSwitchAllCommandClass}. Implements All On/All Off.
+ * @author Pedro Paixao
+ * @since 1.8.0
+ */
+public class ZWaveSwitchAllConverter extends ZWaveCommandClassConverter<ZWaveSwitchAllCommandClass> {
+    private static final Logger logger = LoggerFactory.getLogger(ZWaveSwitchAllConverter.class);
+    private static final int REFRESH_INTERVAL = 0;
+
+    /**
+	 * Constructor. Creates a new instance of the {@link ZWaveSwitchAllConverter} class.
+	 * @param controller the {@link ZWaveController} to use for sending messages.
+	 * @param eventPublisher the {@link EventPublisher} to use to publish events.
+	 */
+    public ZWaveSwitchAllConverter(ZWaveController controller, EventPublisher eventPublisher) {
+        super(controller, eventPublisher);
+        this.addStateConverter(new BinaryDecimalTypeConverter());
+        this.addStateConverter(new BinaryPercentTypeConverter());
+        this.addStateConverter(new IntegerOnOffTypeConverter());
+        this.addStateConverter(new IntegerOpenClosedTypeConverter());
+        this.addStateConverter(new IntegerUpDownTypeConverter());
+        this.addCommandConverter(new BinaryOnOffCommandConverter());
+    }
+
+    /**
+	 * {@inheritDoc}
+	 */
+    @Override
+    public SerialMessage executeRefresh(ZWaveNode node, 
+    		ZWaveSwitchAllCommandClass commandClass, int endpointId, Map<String, String> arguments) {
```

Comment:
Does it really make sense to poll this? IT's not polling the status of the switch, it's polling the status of the configuration (correct?). If so, I don't think it makes sense and I'd probably just remove the polling?
What do you think?